> I am getting errors like
> 
> ```
> ...server\common\src\main\java\org\eclipse\lsp\cobol\common\action\CodeActionProvider.java:19: error: package org.eclipse.lsp4j does not exist
> import org.eclipse.lsp4j.CodeAction;
> ```
> 
> and eventually
> 
> ```
> ...server\common\src\main\java\org\eclipse\lsp\cobol\common\mapping\ExtendedDocument.java:22: error: package org.eclipse.lsp4j does not exist
> import org.eclipse.lsp4j.Position;
>                         ^
> Exception in thread "main" com.sun.tools.javac.processing.AnnotationProcessingError: java.lang.StackOverflowError
>         at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:1033)
>         at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:937)
>         at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1265)
>         at jdk.compiler/com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1380)
>         at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1279)
>         at lombok.delombok.Delombok.delombok(Delombok.java:789)
>         at lombok.delombok.Delombok.main(Delombok.java:326)
>         at lombok.delombok.DelombokApp.runDirectly(DelombokApp.java:176)
>         at lombok.delombok.DelombokApp.runApp(DelombokApp.java:47)
>         at lombok.core.Main.go(Main.java:130)
>         at lombok.core.Main.main(Main.java:43)
> ```

I have checked, this errors do not affect the final delomboked code, it is compilable (if you switch off style check)

--- a/_lombok.config
+++ b/_lombok.config
@@ -1,0 +1,15 @@
+# Copyright (c) 2024 Broadcom.
+# The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Broadcom, Inc. - initial API and implementation
+lombok.anyConstructor.suppressConstructorProperties=true
+lombok.nonNull.exceptionType = IllegalArgumentException
+lombok.log.fieldName = LOG
+lombok.addLombokGeneratedAnnotation = true

--- a/delombok.sh
+++ b/delombok.sh
@@ -1,1 +1,2 @@
-java -jar lombok.jar delombok server -d server-delomboked
+(cd server && mvn clean)
+java -jar lombok.jar delombok server -d server-delomboked -f pretty 

